doc/manual/installing.rst

```diff
@@ -93,17 +93,19 @@ These steps are required to generate bitstream (``.bit``) files, build the MiSoC
         $ git clone --recursive https://github.com/m-labs/misoc
         $ export MSCDIR=~/artiq-dev/misoc # append this line to .bashrc
 
-* Build and flash the bitstream and BIOS by running `from the MiSoC top-level directory` ::
+* Download and install ARTIQ: ::
+
+        $ cd ~/artiq-dev
+        $ git clone https://github.com/m-labs/artiq
+        $ python3 setup.py develop --user
```

Comment:
`./make.py -X ~/artiq-dev/artiq/soc -t artiq_ppro all` ← this imports some modules from artiq and fails